test/unittests/test_async_ticket_audits.py

```diff
@@ -0,0 +1,255 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from aioresponses import aioresponses
```

Comment:
Can we try using `unittest.mock.AsyncMock` instead of third party module `aioresponses`?